source/getting-started/quick-start.md

```diff
@@ -0,0 +1,266 @@
+This guide will teach you how to build a simple app using Ember from
+scratch.
+
+We'll cover these steps:
+
+1. Installing Ember
+2. Creating a new application
+3. Defining a route
+4. Writing a UI component
+5. Building your app to be deployed to production
+
+## Install Ember
+
+You can install Ember with a single command using npm, the Node.js package
+manager. Type this into your terminal:
+
+```sh
+npm install ember-cli -g
+```
+
+Don't have npm? [Learn how to install Node.js and npm here][npm].
+
+[npm]: https://docs.npmjs.com/getting-started/installing-node
+
+## Create a New Application
+
+Once you've installed Ember via npm, you will have access to a new
+`ember` command in your terminal. You can use the `ember new` command to
+create a new application.
+
+```sh
+ember new ember-quickstart
+```
+
+This one command will create a new directory called `ember-quickstart`
+and set up a new Ember application inside of it.  Out of the box, your
+application will include:
+
+* A development server
+* Template compilation
+* CSS preprocessing
+* JavaScript and CSS minification
+* ES2015 features via Babel
+
+By providing everything you need to build production-ready web
+applications in an integrated package, Ember makes starting new projects
+a breeze.
+
+Let's make sure everything is working properly. Open the
+`ember-quickstart` directory in your favorite editor, such as Sublime
+Text or Vim. Once you have your editor open, `cd` into the application
+directory and start the development server by typing:
+
+```sh
+cd ember-quickstart
+ember serve
+```
+
+After a few seconds, you should see output that looks like this:
+
+```text
+Livereload server on http://localhost:49152
+Serving on http://localhost:4200/
+```
+
+(To stop the server at any time, type Ctrl-C in your terminal.)
+
+Open [http://localhost:4200/](http://localhost:4200) in your browser of
+choice. You should see a page that says "Welcome to Ember" and not much
+else. Congratulations! You just created and booted your first Ember app.
+
+Switch to your editor and open `app/templates/application.hbs`. This is
+called the `application` template and it is always on screen while the
+user has your application loaded.
+
+In your editor, change the text inside the `<h1>` from `Welcome to
+Ember` to `PeopleTracker` and save the file. Notice that Ember detects
+the change you just made and automatically reloads the page for you in
+the background. You should see that "Welcome to Ember" has been
+replaced by "PeopleTracker".
+
+## Define a Route
+
+Ember makes URLs the heart of your application, so you never build apps
+that feel broken to your users. Before doing anything in Ember, first think
+about your application's URLs.
```

Comment:
> first think about your application's URLs.

This isn't actionable advice because you haven't helped the reader understand how to think about URLs. If you can't explain it right here, then you should instead try to get them excited about how either this guide will eventually teach it to them, or if you can't teach it to them in this guide, you should try to get them excited about how _using Ember_ will teach it to them. Example: "Working with Ember will help you learn how to think about designing beautiful URL architectures for your apps."